internal/ingress/controller/template/configmap_test.go

```diff
@@ -31,6 +31,24 @@ func TestFilterErrors(t *testing.T) {
 	}
 }
 
+func TestProxytTimeoutParsing(t *testing.T) {
+
+	cfg := ReadConfig(map[string]string{"proxy-protocol-header-timeout": "35s"})
```

Comment:
I'd use a map here in order to remain as consistent as possible with other tests. E.g.

```go
	testCases := map[string]struct {
		input  string
		expect string // or time.Duration, see previous comment
	}{
		"valid duration":   {"35s", "35s" },  // or 35*time.Second
		"invalid duration": {"3zxs", "5s" },  // or 5*time.Second
	}

	for n, tc := range testCases {
		// ...
	}
```